blocked-edges/4.10.11-parallel-ceph_fsync.yaml

```diff
@@ -5,11 +5,4 @@ name: CephParallelFsync
 message: |-
   This update would introduce a CephFS kernel driver regression, exposing a kernel panic when workloads make parallel ceph_fsync calls to the same file.  The update also introduces many bug fixes as described in the errata, so weigh those against the risk of Ceph kernel panics when deciding whether to update or wait for an OpenShift release that also fixes the Ceph regression.
 matchingRules:
-- type: PromQL
-  promql:
-    promql: |
-      topk(1,
-        label_replace(group(ceph_health_status), "ceph", "yes", "", "")
-        or
-        label_replace(0 * group(cluster_version), "ceph", "no", "", "")
```

Comment:
possibly shift this PromQL into the `message`? Or the linked bug (although this one links https://bugzilla.redhat.com/show_bug.cgi?id=2076312#c9, which seems to be private)? But the current message is phrased as if we know (or suspect) the cluster is exposed, while with Always this will also trip for clusters we know are not exposed. Or 🤷, maybe that's more polish than we care about for such an old 4.10.z target as these.

Reply:
I'd prefer to just leave it as is. Only 17% of the 4.10 fleet has upgrades to < 4.10.17 and those all have paths to better edges listed more prominently.